plugins/optimization-detective/docs/hooks.md

```diff
@@ -367,3 +367,19 @@ The ETag is a unique identifier that changes whenever the underlying data used t
 6. The list of active plugins.
 
 A change in ETag means that any previously-collected URL Metrics will be immediately considered stale. When the ETag for URL Metrics in a complete viewport group no longer matches the current environment's ETag, new URL Metrics will then begin to be collected until there are no more stored URL Metrics with the old ETag.
+
+### Filter: `od_url_metric_garbage_collection_ttl` (default: 3 * MONTH_IN_SECONDS)
+
+Filters the expiration time (TTL) after which an unmodified od_url_metrics post not touched since that time will be garbage collected.
```

Comment:
```suggestion
Filters the expiration time (TTL) after which an unmodified `od_url_metrics` post not touched since that time will be garbage collected.
```